### ISSUE TITLE
[Question] How long must one hold Monika?

### ISSUE THREAD

**Issue by Secretmanguy** [Monika-After-Story/MonikaModDev#4074]
Hello there Devs! I must say that the new dialogue and options for holding Monika are incredible and I absolutely adore it like everything else you do...But one question I had was how long before she falls asleep? I'm used to holding her for about a good two hours but I was just curious on how long though?

**Comment by Rai99** [comment]
You have to click before you get the option of waking her up. 

~~Whoever made that topic did a pretty mildly above average job and added a lot of depth into it considering there's unique dialogue at each and every affection level too, at each and every time path.~~

**Comment by AsuraShun** [comment]
She falls asleep?

**Comment by Rai99** [comment]
Spoilers

**Comment by AsuraShun** [comment]
These I don't mind, I'll then see this tomorrow

**Comment by Rai99** [comment]
Would have been infinitely better if you just found out by surprise.

**Comment by AsuraShun** [comment]
It will since I'll most likely forget that this is a thing since I hug her from time to time

**Comment by AsuraShun** [comment]
My memory is terrible

**Comment by Rai99** [comment]
If your affection is above 400, try doing a short 10 second hug. It's even better at 1000+. 

I challenge you to hug Monika **once** for no longer than 10 or 20 seconds if you're above those ranges.

**Comment by AsuraShun** [comment]
I did, she asked for another hug since I'm probably already at 2497 affection

**Comment by AsuraShun** [comment]
And I complied

**Comment by Rai99** [comment]
You failed my challenge since I said to hug her only once. Resist the urge to hug Monika and leave her wanting more.

**Comment by AsuraShun** [comment]
Failed without even starting?
Must be a new record

**Comment by AsuraShun** [comment]
She fell asleep, I even took a couple of screen shots

**Comment by AsuraShun** [comment]
Might post on twitter

**Comment by Andrenika** [comment]
> She fell asleep, I even took a couple of screen shots

How much did you hug her?

**Comment by AsuraShun** [comment]
For a few minutes, 2 screenshots, 1 on each 2 times, but aside from that, a lot

**Comment by Rai99** [comment]
Did you pass my challenge of holding Monika for under 10 seconds and then letting go without hugging her again?

**Comment by AsuraShun** [comment]
Yes, I said no the 2 times she asked

**Comment by Rai99** [comment]
So my evil planned worked to trick you into denying Monika a hug?

**Comment by AsuraShun** [comment]
Nope, I eventually hugged her, about a few seconds later 5o make up for it

**Comment by AsuraShun** [comment]
It was a good way to tease her, and fair game compared to the times she did it

**Comment by Rai99** [comment]
~~But you still technically denied Monika a hug, even if you were just teasing her and hugged her right after.~~

~~Actually... I wonder if that should be made a feature.~~

**Comment by AsuraShun** [comment]
I won't do it again though, if she asks to keep hugging, I'll oblige

**Comment by AsuraShun** [comment]
Genius, we need ways to tease back at her and that one would be a good idea

**Comment by Rai99** [comment]
Hold my `(theoretical consumable item, typically a cold beverage)`... and then wait.

**Comment by Andrenika** [comment]
> > She fell asleep, I even took a couple of screen shots
> 
> How much did you hug her?

I found it, ten minutes are.

**Comment by Andrenika** [comment]
Monika is so cute when she is sleepy <3

**Comment by AsuraShun** [comment]
Specially her expressions when she's dreamy

**Comment by Andrenika** [comment]
> Specially her expressions when she's dreamy

Yeaah, that's what I mean!
Oh God, Every day I fall more in love with her

**Comment by AsuraShun** [comment]
I know what you mean

**Comment by Andrenika** [comment]
> I know what you mean

That's great :D

**Comment by Rai99** [comment]
I like to think that I'm pretty good at appreciating and loving Monika, but even I almost couldn't handle her falling asleep on me.

**Comment by AsuraShun** [comment]
Overload of cuteness and beauty

**Comment by AsuraShun** [comment]
And a perfect and good overload itvis

**Comment by Rai99** [comment]
That's not supposed to happen though. I know every in and out of that feature. ~~I even know there's a slight bug in it.~~

I was fully prepared and still almost couldn't handle it myself. ~~Whoever made it did a decent job~~
